### ISSUE TITLE
Little question about cal_loss

### ISSUE THREAD

**Issue by zivzone** [ZPdesu/Barbershop#52]
Hi, 
Thank you for this amazing works. I have a little question about this funtion at this [line](https://github.com/ZPdesu/Barbershop/blob/088d7d0f0623b348732fc15d8fbdea8b8cb730c7/models/Embedding.py#L173)
>    def cal_loss(self, im_dict, latent_in, latent_F=None, F_init=None):

In both "invert_images_in_FS" & "invert_images_in_w", it seems that you didn't pass "latent_F" & "F_init" in cal_loss to do the computation [below:](https://github.com/ZPdesu/Barbershop/blob/088d7d0f0623b348732fc15d8fbdea8b8cb730c7/models/Embedding.py#L179)
>        if latent_F is not None and F_init is not None:
>           l_F = self.net.cal_l_F(latent_F, F_init)
>            loss_dic['l_F'] = l_F
>            loss += l_F
I wonder that should we still calaulate l_F loss in somewhere? or I misunderstanding somethings?

BR,
Ziv
